buildpackage/config_reader.go

```diff
@@ -16,6 +16,7 @@ import (
 type Config struct {
 	Buildpack    dist.BuildpackURI `toml:"buildpack"`
 	Dependencies []dist.ImageOrURI `toml:"dependencies"`
+	Platform     dist.Platform     `toml:"platform"`
```

Comment:
Do you think we should have some extra validation that the contents of the `platform.os` is only either `windows` or `linux`, and anything else produces an error?